_source/_posts/2019-05-31-simple-auth-express-fifteen-minutes.md

```diff
@@ -77,29 +79,45 @@ Express allows for a templating engine. You already set up Handlebars (hbs) abov
 
 **views/layout.hbs**
 {%raw%}
+
```

Comment:
Does adding this line break change the output somehow or is it just for readability?